mws/mws.py

```diff
@@ -229,6 +236,13 @@ def make_request(self, extra_data, method="GET", **kwargs):
         parsed_response.response = response
         return parsed_response
 
+    def set_proxies(self):
```

Comment:
Yes I'll add a note to the contribution docs saying not to worry about codecov (for now)